test/e2e/framework.go

```diff
@@ -96,6 +98,7 @@ const (
 
 	agnhostImage        = "gcr.io/kubernetes-e2e-test-images/agnhost:2.8"
 	busyboxImage        = "projects.registry.vmware.com/library/busybox"
+	busyboxWindowsImage = "projects.registry.vmware.com/antrea/e2eteam-busybox:1.29-windows-amd64-1809"
```

Comment:
is there a multi-arch image we can use to make things simpler? I think we should switch to using the `agnhost` image as much as possible, would this work here?